### PR TITLE
AC-599: Remove Redundant type cast

### PR DESCRIPTION
**Pull request by Abdelaty**
Signed-off-by: abdelaty <abdelaty.mohammedmagdi@gmail.com>

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-599

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [ ] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by Abdelaty** [comment]
@f4ww4z  Hey my friend, Can you review it?

**Comment by Abdelaty** [comment]
@f4ww4z @deepak140596  Hey Friends, Can you help me with this errors?
it's the first PR From my fork. And I can't figure out why it's failed?

**Comment by HerbertYiga** [comment]
CI build fails here!

File: gradle.properties [hunk 1]
```diff
@@ -18,4 +18,3 @@ org.gradle.jvmargs=-Xmx1536m
 
 android.useAndroidX=true
 android.enableJetifier=true
-android.enableSeparateAnnotationProcessing=true
```
Comment (f4ww4z) [inline gradle.properties]: Why did you remove this line?

Comment (Abdelaty) [reply gradle.properties]: It cause failure while building gradle on my device, so I removed it, after I solved the problem of Travis CI, it works now.
Thanks @f4ww4z 
I will remove the PR and submit another one which containing one commit.

**Comment by Abdelaty** [comment]
> CI build fails here!

Yes I see, Thank you @HerbertYiga
